src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_help.py

```diff
@@ -0,0 +1,175 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['bot'] = """
+    type: group
+    short-summary: Manage Bot Services.
+"""
+helps['bot create'] = """
+    type: command
+    short-summary: Create a new Bot Service.
+"""
+helps['bot show'] = """
+    type: command
+    short-summary: Get an existing Bot Service.
+"""
+helps['bot prepare-publish'] = """
+    type: command
+    short-summary: Add azure specific scripts to your local source code directory to
+                   be able to publish back using az bot publish.
+"""
+helps['bot delete'] = """
+    type: command
+    short-summary: delete an existing Bot Service.
+"""
+helps['bot update'] = """
+    type: command
+    short-summary: update an existing Bot Service.
+    examples:
+        - name: Update description on a bot
+          text: |-
+            az bot update -n botName -g MyResourceGroup --set properties.description="some description"
+"""
+helps['bot publish'] = """
+    type: command
+    short-summary: publish to an existing Bot Service.
+    long-summary: publish your github or zip code to a bot service.
+                  This lets you overwrite the existing template code on a bot.
```

Comment:
removed 'existing' to avoid confusion. also made the description more clearer since the actual command has now been modified to take just the folder path and not a zip or git repo